ocs_ci/helpers/helpers.py

```diff
@@ -1392,6 +1392,55 @@ def measure_pvc_creation_time_bulk(interface, pvc_name_list, wait_time=60):
     return pvc_dict
 
 
+def measure_multiple_pvc_deletion_time(interface, pvc_name, status):
+    """
+    Get the starting/ending deletion time of a PVC based on provisioner logs
+
+    Args:
+        interface (str): The interface backed the PVC
+        pvc_name (str / list): Name of the PVC(s) for creation time
+                               the list will be list of pvc objects
+        status (str): the status that we want to get - Start / End
+
+    Returns:
+        datetime object: Time of PVC(s) deletion
+
+    """
+    # Define the status that need to retrieve
+    operation = "started"
+    if status.lower() == "end":
+        operation = "succeeded"
+
+    format = "%H:%M:%S.%f"
+    # Get the correct provisioner pod based on the interface
+    pod_name = pod.get_csi_provisioner_pod(interface)
+    # due to some delay in CSI log generation added wait
+    time.sleep(60)
```

Comment:
Would recommend to change the sleep time according to the PVC numbers. Have faced this problem with just 100 PVC 60sec might be enough but with more PVC numbers like 500/1000 this 60sec won't be enough.

Reply:
Is it good to create separate function to address this?

Reply:
Addressed this conversation